distribution/lib/Standard/Test/0.0.0-dev/src/Extensions.enso

```diff
@@ -295,7 +295,7 @@ Error.should_equal self that frames_to_skip=0 =
 Number.should_equal : Float -> Float -> Integer -> Spec_Result
 Number.should_equal self that epsilon=0 frames_to_skip=0 =
     matches = case that of
-        _ : Number -> self.equals that epsilon
+        n : Number -> self.equals n epsilon
```

Comment:
This shows a small nuance. Previously there was no difference between `that` and `n`. Now there is as 571cce6 documents.